doc/May-2022-Upgrade.md

```diff
@@ -0,0 +1,634 @@
+    ```
+    
+1. Install PostgreSQL 14
+
+    ```shell
+    root@postgres:/# apt-get update
+    root@postgres:/# apt-cache policy postgresql-14-postgis-3
+    root@postgres:/# apt-cache policy postgis
+    ```
+
+    _Store the PostGIS version in a variable to use later_
+    
+    ```shell
+    root@postgres:/# POSTGIS_VERSION_14=$(apt-cache policy postgresql-14-postgis-3|grep Candidate:|awk '{print $2}')
+    root@postgres:/# apt-get install -y --no-install-recommends postgresql-14-postgis-3=${POSTGIS_VERSION_14} postgresql-14-postgis-3-scripts=${POSTGIS_VERSION_14} postgis postgresql-contrib-14
+    root@postgres:/# apt-get upgrade
+    ```
+
+1. Init DB
+
+    ```shell
+    root@postgres:/# chown -R postgres:postgres /var/lib/postgresql/data14/
+    root@postgres:/# su - postgres -c "/usr/lib/postgresql/14/bin/initdb -U $POSTGRES_USER --encoding=utf8 --locale=en_US.utf-8 -D /var/lib/postgresql/data14/"
+    ```
+    Results should look like this:
+
+    > ```
+    > Success. You can now start the database server using:
+    >      /usr/lib/postgresql/14/bin/pg_ctl -D /var/lib/postgresql/data14/ -l logfile start
+    > ```
+
+1. Start PostgreSQL 14 to ensure database has been initialized successfully
+
+    ```shell
+    root@postgres:/# su - postgres -c '/usr/lib/postgresql/14/bin/pg_ctl -D /var/lib/postgresql/data14/ start'
+    ```
+    > ```
+    > ...
+    > LOG:  database system is ready to accept connections
+    > ```
+
+    Press `enter` to go back to prompt.
+
+1. Stop the server
+
+    ```
+    root@postgres:/# su - postgres -c '/usr/lib/postgresql/14/bin/pg_ctl -D /var/lib/postgresql/data14/ stop -m fast'
+    ```
+
+    > ```
+    > ...
+    > server stopped
+    > ```
+
+
+1. Upgrade Postgres 9.5
+
+    ```shell
+    root@postgres:/# apt-cache policy postgresql-9.5-postgis-3
+    root@postgres:/# POSTGIS_VERSION_9_5=$(apt-cache policy postgresql-9.5-postgis-3|grep Candidate:|awk '{print $2}')
+    root@postgres:/# apt-get install -y --no-install-recommends postgresql-9.5-postgis-3=${POSTGIS_VERSION_9_5} postgresql-9.5-postgis-3-scripts=${POSTGIS_VERSION_9_5}
+    root@postgres:/# apt-get upgrade
+    ```
+ 
+1. Start PostgreSQL 9.5
+
+    ```shell
+    root@postgres:/# su - postgres -c '/usr/lib/postgresql/9.5/bin/pg_ctl -D /var/lib/postgresql/data/ start'
+    ```
+    
+    Press `enter` to go back to prompt and enter `psql` cli client:
+    
+    ```shell
+    root@postgres:/# /usr/lib/postgresql/9.5/bin/psql -U $POSTGRES_USER -d postgres
+    ```
+
+1. Upgrade PostGIS extension
+
+    You may see some warnings `WARNING:  'postgis.backend' is already set and cannot be changed until you reconnect`. That's ok, you can keep going ahead.
+
+    Depending on your kobo-docker environment, databases may have other names.  
+    You may need to adapt the snippet below to your curren configuration.
+    
+    _Notes: You may need to copy lines below one by one because sometimes copying the whole block does not work as expected._
```

Comment:
This was actually only the case for the `\c ...;` commands — once I connected to the new db then I could copy the block of SQL without issue. But the note here is fine 👌